### ISSUE TITLE
[Chore] Gitflow can be improved

### ISSUE THREAD

**Issue by longnd** [aungkoko1234/web-scrapper-backend#15]
Issue
---
While the 2 applications use pull requests to separate the work, there are room for improvement
- The PRs' titles are usually not descriptive to give an idea of what it delivers.  e.g. `Feature/auth`, `Develop`. Luckily some PRs have short descriptions so they're understandable at first glance.
- They also do not follow a consistent naming pattern, e.g. `Feature/auth`, `chore - fix test for component`
- Some PRs handle multiple responsibilities, e.g. https://github.com/aungkoko1234/web-scrapper-backend/pull/4/files handles both file upload and searching keywords using queue. Ideally, each PR should responsible for a single feature only.

Expected
--
- Proper [git-flow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) workflow usage.

**Comment by aungkoko1234** [comment]
Yeah, git flow can also be improved.  There is also unrelated files in some PR. My bad. I will take care and follow the standard  more in the future. Thanks for your guidance. For this project, how can I resolve this issue.

**Comment by longnd** [comment]
> how can I resolve this issue.

if you decide to open some PRs to fix the opened issues, please use proper git flow :)

**Comment by aungkoko1234** [comment]
Noted

**Comment by longnd** [comment]
may I know why the new PRs that fixed some issues were merged to the `main` branch instead the `develop` branch?
